File: 백준/Silver/2563. 색종이/색종이.py
n = int(input())

ans = 0

arr = [[0] * 101 for i in range(101)]

for _ in range(n):
    y,x = map(int, input().split())

    for i in range(x, x+10):
        for j in range(y, y+10):
            arr[i][j] += 1

for i in range(101):
    for j in range(101):
        if arr[i][j] != 0:
            ans += 1
print(ans)

File: 백준/Silver/2563. 색종이/색종이.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {


    val n = readLine().toInt()

    var ans = 0

    var arr = Array(n) {IntArray(2) {0} }

    repeat(n) {
        val st = StringTokenizer(readLine())

        arr[it][0] = st.nextToken().toInt()
        arr[it][1] = st.nextToken().toInt()

    }

    var check = Array(100) {IntArray(100) {0} }

    for (i in arr.indices) {
        var x = arr[i][0]
        var y = arr[i][1]

        for (j in x until x + 10) {
            for (k in y until y + 10) {
                if (check[j][k] == 0) check[j][k] = 1
            }
        }
    }

    for (i in 0 until 100) {
        for (j in 0 until 100) {
            if (check[i][j] == 1) ans++;
        }
    }

    println(ans)









}